content/02.introduction.md

```diff
@@ -1,41 +1,52 @@
+This strong code-dependent workflow imposes steep requirements towards an efficient, well-documented process in the publication and collaboration of maintenaible scientific code [@doi:10.1038/d41586-020-02462-7].
+To facilitate this process, scientists have been increasingly borrowing and adopting tools from  information system technology, such as cloud-based services for documentation and version control [_e.g._ from the Google Suite (with Docs, Sheets and Drive), the Microsoft Suite (with Word, Excel and OneDrive), and Github] [@doi: 10.1038/538127a].
+However, most researchers lack exposure to adequate software development practices and are required to dedicate valuable time and effort to self-teach the use of research-faciliting tools, and thus may find practical barriers when applying adequate standards in the maintenance of their scientific code [@doi:10.1145/2063348.2063374; @doi:10.1371/journal.pbio.1001745; @doi:10.1109/SECSE.2009.5069155].
+Here, we review and discuss one of the most used web-based platforms for computational version control and collaboration, Github, and provide researchers in ecology and evolutionary biology with practical workflows aiming at facilitating their scientific code and management process.
 
-Researchers in ecology and evolutionary biology (EEB) are starting to collaborate on software as part of their research, and some are interacting with GitHub for the first time [@doi:10.1038/s41559-017-0160].
-For first-time users, the GitHub learning curve can seem overwhelming because the creation of the platform and its features were initially centered on collaboration for software development [@doi:10.1109/ASONAM.2016.7752419].
-However, by leveraging existing tools on GitHub and the wide range of collaborations they can enable, researchers in EEB can make the most out of their collaborative projects.
+### General background about GitHub and Git
 
-### What's already been written about GitHub
+*Contributors to this section: RCO, SSHS, PHPB*
+
+With over 73 million registered users, GitHub became the most common web-platform used for documentation and collaboration on computer code [@https://github.com/about].
+[GitHub](https://github.com/) combines [Git](https://git-scm.com/), its underlying version control system, to provide less complicated, yet powerful services allowing users to participate in projects, contribute, report and discuss code issues, and search for existing code as well as publish their own code.
+Through its version control, users have a detailed, chronological record on the files and directories stored in their repositories [@doi:10.1080/00031305.2017.1399928].
+This workflow provides a strong and clear advantage over receiving, processing and sending files back-and-forth, a process that can easily become challenging and time-consuming in projects extending in time and in the number of collaborators [@doi:10.1186/1751-0473-8-7].
```

Comment:
I think maybe somewhere around here we need a sentence explaining that Github (and git) are tools built with software developers in mind, but that many of the benefits of these tools apply to EEB researchers.

Reply:
This information was incorporated in the following paragraph, which begins on line 34, with "_The voluminous user-community and the numerous resources..._". 

This paragraph addresses the lack of domain-specific resources for ecology and evolutionary biology researchers. 

The following sentence (in line 36) discusses the point you mention: 
> "Because GitHub and its features have been created centered on collaboration for software development in information systems [@doi:10.1109/ASONAM.2016.7752419], first-time users from domains without formal training in information technology can face challenging learning curves."